### DISCUSSION TITLE
What name should the signify-browser-extension have in the Chrome, Firefox, etc. catalogs?

### DISCUSSION THREAD

**Discussion by 2byrds** [WebOfTrust/signify-browser-extension#15] [Polls]
Likely "signify-browser-extension" is too verbose for the browser extension catalogs. We discussed that in [a previous poll](https://github.com/WebOfTrust/signify-browser-extension/discussions/1), regarding the repo name.
Lets focus on the public catalog name for the extension.

You can search the catalogs to see how a name overlaps here, by replacing "<NameOfExtension>":
https://addons.mozilla.org/en-US/firefox/search/?q=<NameOfExtension>&type=extension
https://chromewebstore.google.com/search/<NameOfExtension>?itemTypes=EXTENSION

If you think of a name not on this list, post in the comments the name and what advantage the name has over the previous suggestions.

Some things to consider:

- Is it unique?
  - Does the name already exist?
  -  Will it be confused for similar extensions?
- Is it short/memorable?
- Does it tie back to "Signify" in some way?

**Comment by dhh1128** [comment]
I want to vote -1 on signify, for the reason @2byrds already noted. I like the first 4 options. +0 on "sentry" because I think it's boring/forgettable.
